examples/release_highlights/plot_release_highlights_1_2_0.py

```diff
@@ -82,6 +82,32 @@
 )
 X.head()
 
+# %%
+# New and enhanced displays
+# -------------------------
+# The :class:`~metrics.PredictionErrorDisplay` provides a one way to analyze
+# regression models in a qualitative manner.
+import matplotlib.pyplot as plt
+from sklearn.metrics import PredictionErrorDisplay
+
+X, y = load_diabetes(return_X_y=True, as_frame=True)
+fig, axs = plt.subplots(nrows=1, ncols=2, figsize=(12, 5))
+PredictionErrorDisplay.from_estimator(
+    hist_no_interact, X, y, kind="actual_vs_predicted", ax=axs[0]
+)
+PredictionErrorDisplay.from_estimator(
+    hist_no_interact, X, y, kind="residual_vs_predicted", ax=axs[1]
+)
+
+# %%
+# The :class:`~model_selection.LearningCurveDisplay` is now available to plot
```

Comment:
```suggestion
# :class:`~model_selection.LearningCurveDisplay` is now available to plot
```

Either "The classname class" followed by the verb or "classname" followed by the verbe. "The" followed by a proper noun does not sound grammatically correct to my ear.